boa/src/builtins/bigint/mod.rs

```diff
@@ -64,6 +64,18 @@ impl BigInt {
         Ok(data)
     }
 
+    #[inline]
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
Whats this clippy allow for?

Reply:
It's because clippy says that `to_`, `as_` functions should take a reference to `Self`

Reply:
I will change this when I make `ToBigInt` (to_bigint) spec compliant, but for no this will suffice